metadata.json

```diff
@@ -48,6 +48,10 @@
       "name": "puppet/selinux",
       "version_requirement": ">= 1.1.0 < 2.0.0"
     }
+    {
+      "name": "chocolatey-chocolatey",
```

Comment:
In my opinion this is a optional dependency and should not be added to the metadata.json. Can you please document this in the README.md?